packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -30,6 +30,14 @@ class CreateCommand extends Command {
       defaultsTo: true,
       help: 'Whether to run "pub get" after the project has been created.'
     );
+    argParser.addFlag(
+      'with-driver-test',
+      negatable: true,
+      defaultsTo: false,
+      help:
+        'Also add Flutter Driver dependencies and generate a sample driver '
```

Comment:
Probably just one line here for the string; there's no 80 col limit, just whatever works for that particular line.